source/fuzz/fuzzer_pass_obfuscate_constants.cpp

```diff
@@ -258,11 +260,10 @@ void FuzzerPassObfuscateConstants::ObfuscateBoolConstant(
   auto chosen_type_id =
       available_types_with_uniforms[GetFuzzerContext()->RandomIndex(
           available_types_with_uniforms)];
-  auto available_constants = GetTransformationContext()
-                                 ->GetFactManager()
-                                 ->GetConstantsAvailableFromUniformsForType(
-                                     GetIRContext(), chosen_type_id);
-  if (available_constants.size() == 1) {
+  auto available_constant_words =
+      fuzzerutil::GetConstantWordsFromUniformsForType(
+          GetTransformationContext(), chosen_type_id);
```

Comment:
I'm unsure about the motivation for adding a new method to fuzzerutil for this purpose.  It appears that this is the only place from which we call the new method.

Can we discuss that a bit here?

Reply:
I actually have no particular reason why I placed it in `fuzzerutil`. I've just thought that it may be useful to someone else. We can make it a private method in `FuzzerPassObfuscateConstants`.

Reply:
@Vasniktel That sounds good to me - please go ahead and we can move it if we do later find another customer for it.

Reply:
Fixed in 3a7f116.